### PR TITLE
Add Puerto Rico 12

### PR DESCRIPTION
**Pull request by tanyh2018**

### CODE REVIEW

File: _posts/2018-10-25-PR12.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+<img src="/images/pr_20181025.jpg" width="400"/>
+</a>
+
+**Organizer: Tanyh** 
+
+We played our 12th Puerto Rico today. Here is the score: 
+
+| Team                | Building | Victory | Additional | Total |
+| --------------------| -------- | ------- | ---------- | ----- |
+| Gushan              | 15       | 40      | 13         | 68    |
```
Comment (xshi) [inline _posts/2018-10-25-PR12.md]: Gushan -> Shan :)

File: _posts/2018-10-25-PR12.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+| Yuhang              | 18       | 27      | 5          | 50    |
+| Amit Ryuta          | 12       | 22      | 0          | 34    |
```
Comment (xshi) [inline _posts/2018-10-25-PR12.md]: Amit, Ryuta

File: _posts/2018-10-25-PR12.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+date: 2018-10-25 11:30:00 +0800
+author: tanyh2018
+categories: pr
+---
+
+<a href="/images/pr_20181025.jpg">
+<img src="/images/pr_20181025.jpg" width="400"/>
+</a>
+
+**Organizer: Tanyh**
```
Comment (xshi) [inline _posts/2018-10-25-PR12.md]: Tanyh -> Yuhang

Comment (tanyh2018) [reply _posts/2018-10-25-PR12.md]: I have modified these.

**Pull request merged**